src/utils/events/move-input.js

```diff
@@ -24,9 +26,29 @@ export default class MoveInput {
   }
 
   handler(event) {
-    this.callback({
-      srcEvent: event,
-      target: this.element
-    });
+    switch (event.type) {
+    case 'mousedown':
+      if (event.button === 0) {
+        this.pressed = true;
+      }
+      break;
+    case 'mousemove':
+      if (event.which !== 1) {
+        this.pressed = false;
+      }
+      if (!this.pressed) {
+        // Drag events are emitted by hammer already, we just need the hover event
```

Comment:
Among all move events we have drag (button down) and hover (button up). The point here is that the move events with button down is already emitted. And this custom input class should only fire move events when the button is up.